pkgs/by-name/gi/git-of-theseus/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+python3Packages.buildPythonApplication {
+  pname = "git-of-theseus";
+  version = "0.3.4";
```

Comment:
needs to be `version-unstable-date`. Same thing for PR title.

Reply:
Whoops, thank you for flagging. Fixed it

Reply:
The version must start with a digit as can be read in [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning), the common pattern in this case being 0.3.4-unstable-2023-11-25